en/docs/tutorials/configure-automatic-deployment.md

```diff
@@ -0,0 +1,108 @@
+# Configure Automatic Deployment
+
```

Comment:
Can we check if we can convert the full sized images to three quarters or half?

Reply:
Changed to three quarters